_content/monographs/lost-kingdoms-hindu-buddhist-sculpture_guy-john.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Lost Kingdoms: Hindu-Buddhist Sculpture of Early Southeast Asia"
+editor: "John Guy"
+external_url: "https://www.metmuseum.org/art/metpublications/Lost_Kingdoms_Hindu_Buddhist_Sculpture_of_Early_Southeast_Asia"
+drive_links:
+  - "https://drive.google.com/file/d/1L-tTixVH9Qgp87bP51wTECt1xWR72bzv/view?usp=sharing"
+course: theravada-roots
+tags:
+  - bart
+  - southeast-asia
+year: 2014
+olid: OL38368022M
+oclc: 871203082
+publisher: "The Metropolitan Museum of Art"
+address: "New York"
+pages: 270
```

Comment:
I usually include the notes and bibliography in the page count of academic papers, so for consistency let's say 305 here.  Yes it slightly pads things out, but since academic writing is a bit harder to read, I think it's a fair trade